src/controllers/controllermanager.cpp

```diff
@@ -37,6 +37,12 @@ const int kPollIntervalMillis = 5;
 const int kPollIntervalMillis = 1;
 #endif
 
+// Note: A standard Midi device runs at 31.25 kbps,
+// a usual Midi message has 30 bits so we may not expect more than
+// 1000 messages per second
+// PortMidiController uses a buffer of 64 messages which results in
+// Maximum polling interval of 64 ms a theoritical minimum of 1 ms
+
```

Comment:
This will only heal one minor symptom. The real issue is that the process can stuck  in the loop during a Mixxx run. It is a high priority thread, which consumes so much CPU that Mixxx is unusable, It does bitchy except GUI commands and it also does not passes midi telegrams to the controller. 

So we need to have an idea when we can exit the loop without loosing telegrams. If it turns out that this patch has issues, we should increase the maximum burst size from 64 to an other value instead.  

I am curious to here test results from Pegasus high res jog.